packages/account/src/BiconomySmartAccount.ts

```diff
@@ -184,7 +202,7 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
     this.initCode = ethers.utils.hexConcat([
       this.factory.address,
       this.factory.interface.encodeFunctionData('deployCounterFactualAccount', [
-        await this.signer.getAddress(),
+        this.owner,
```

Comment:
setInitCode will always be called after setting intended final owner as this.owner state?

Reply:
yeah it is

Reply:
ok